userspace/libpman/src/stats.c

```diff
@@ -289,7 +296,6 @@ struct metrics_v2 *pman_get_metrics_v2(uint32_t flags, uint32_t *nstats, int32_t
 				{
 				case RUN_CNT:
 					strlcat(g_state.stats[offset].name, modern_bpf_libbpf_stats_names[RUN_CNT], sizeof(g_state.stats[offset].name));
-					g_state.stats[stat].flags = METRICS_V2_KERNEL_COUNTERS;
```

Comment:
this was a bug

Reply:
Is it intentional to have differing indexing `g_state.stats[stat].` and `g_state.stats[offset].`?

Reply:
nope it was a bug! great catch!